corehq/motech/repeaters/management/commands/populate_repeater_names.py

```diff
@@ -0,0 +1,30 @@
+from django.core.management.base import BaseCommand
+from corehq.motech.repeaters.models import SQLRepeater
+
+CHUNKSIZE = 1
```

Comment:
is this meant to be 1?

Reply:
Ah, good catch, it was 1 for testing purposes. I'll update